addons/mail/models/mail_channel.py

```diff
@@ -1133,3 +1133,13 @@ def _execute_command_who(self, **kwargs):
             msg = _("Users in this channel: %(members)s %(dots)s and you.", members=", ".join(members), dots=dots)
 
         self._send_transient_message(partner, msg)
+
+    @api.model
+    def _mark_seen_general_channel(self):
```

Comment:
I would use the technical name of the channel here, and specify it's a mark as seen for everyone because that is very unusual and specific.
```suggestion
    def _mark_channel_all_employees_as_seen_for_everyone(self):
```